plugins/legion-hid2/fu-legion-hid2.rs

```diff
@@ -3,65 +3,65 @@
 
 // Commands sent ove rXInput, DInput
 #[repr(u32)]
-enum FuLegionHid2Command {
+enum LegionHid2Command {
     GetVersion = 0x01,
     GetMcuId = 0x02,
     StartIap = 0xE1,
     IcReset = 0xEF,
 }
 
 #[repr(u8)]
-enum FuLegionHid2ReportId {
+enum LegionHid2ReportId {
```

Comment:
You don't need to do this since d69307979affd77624c31dd932370834b57b3c08

Reply:
In 1.9.x?

Reply:
Yes, it's how I did the jabra-file backport.

Reply:
Ah I see, thanks!  Fixed.